java/email-contact-form/src/Cors.java

```diff
@@ -0,0 +1,55 @@
+package io.openruntimes.java.src;
+
+import io.openruntimes.java.RuntimeContext;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+import java.lang.*;
```

Comment:
I don't think this is required. Everything in the java.lang package is imported by default.

Reply:
Yup. Removed the unwanted import statement